src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithDefaultNullFW.java

```diff
@@ -0,0 +1,387 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import static org.agrona.BitUtil.SIZE_OF_BYTE;
+
+import java.text.MessageFormat;
+
+import org.agrona.BitUtil;
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.reaktivity.reaktor.internal.test.types.Flyweight;
+import org.reaktivity.reaktor.internal.test.types.inner.VariantEnumKindOfUint32FW;
+import org.reaktivity.reaktor.internal.test.types.inner.VariantEnumKindWithInt32FW;
+import org.reaktivity.reaktor.internal.test.types.inner.VariantEnumKindWithString32FW;
+
+public class ListWithDefaultNullFW extends Flyweight
+{
+    private static final int PHYSICAL_LENGTH_OFFSET = 0;
+
+    private static final int PHYSICAL_LENGTH_SIZE = BitUtil.SIZE_OF_INT;
+
+    private static final int LOGICAL_LENGTH_OFFSET = PHYSICAL_LENGTH_OFFSET + PHYSICAL_LENGTH_SIZE;
+
+    private static final int LOGICAL_LENGTH_SIZE = BitUtil.SIZE_OF_INT;
+
+    private static final byte NULL_VALUE = 0x40;
+
+    private static final int FIRST_FIELD_OFFSET = LOGICAL_LENGTH_OFFSET + LOGICAL_LENGTH_SIZE;
+
+    private static final int INDEX_VARIANT_OF_STRING1 = 0;
+
+    private static final int INDEX_VARIANT_OF_STRING2 = 1;
+
+    private static final int MASK_VARIANT_OF_STRING2 = 1 << INDEX_VARIANT_OF_STRING2;
+
+    private static final int INDEX_VARIANT_OF_UINT = 2;
+
+    private static final int MASK_VARIANT_OF_UINT = 1 << INDEX_VARIANT_OF_UINT;
```

Comment:
`bitmask` is `long`, so `MASK_xxx` should also be `long` and comparison with `0` should be `0L`